lib/utils.pm

```diff
@@ -1246,15 +1246,7 @@ helper function to restart network
 
 sub restart_network {
     if (is_qemu && systemctl('is-active NetworkManager', ignore_failure => 1) == 0) {
-        my $state = check_nm_connectivity(1);
-
-        if (!($state =~ /full/)) {
-            systemctl('restart NetworkManager');
```

Comment:
I don't understand. If we remove conditional restart here, where it will be unconditionally force restarted?    I don't see force restart anywhere. Is commit message right?

Reply:
During my test, systemctl('restart NetworkManager') can not disconnect / connect dev,  the "real" restart which can make dev connect state change is using nmcli.

Reply:
Yes, but commit message says something else. It says `Force unconditional restart`, but i don't understand where it is force restarted. Or you wanted to say `Remove conditional restart of NetworkManager ` ?

Reply:
Please change it to `ipsec: Refactor network restart`, just to have it more clean.

Indeed, i was confused with commit message.